client/src/routing/AuthCheck.js

```diff
@@ -23,11 +25,14 @@ const AuthCheck = () => {
   return (
     <Router>
       <Navbar />
+      <div className='navbar-offset'>
       <Alerts />
+      <DeleteModal />
```

Comment:
do you have to put these here? It feels weird that you have things that are not Routes or PrivateRoutes as direct children of your Router